app/controllers/admin/deduplicate_patients_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::DeduplicatePatientsController < AdminController
+  skip_before_action :verify_authenticity_token
+  DUPLICATE_LIMIT = 250
+
+  def show
+    authorize { current_admin.power_user? }
+
+    duplicate_patient_ids = PatientDeduplication::Strategies.identifier_match(limit: DUPLICATE_LIMIT)
+    @duplicate_count = duplicate_patient_ids.count
+    @patients = Patient.where(id: duplicate_patient_ids.sample)
+  end
+
+  def merge
+    authorize { current_admin.power_user? }
+
+    duplicate_patients = Patient.where(id: params[:duplicate_patients])
+    PatientDeduplication::Deduplicator.new(duplicate_patients, user: current_admin).merge
```

Comment:
This is going to do a bunch of work in the scope of a request, especially if the dupe patients have many BP's or encounters. We probably want to move this to a background job.

Reply:
We think this is an ok trade-off to keep the process simple. We've seen most merges happen within a few seconds (even large ones) and the we would also like to give the user feedback about the merge actually succeeding/failing.

Reply:
Given it's only 1 patient group at a time, I guess that's ok. The complexity only scales with how _long_ the two dupes have been around, and how many BPs they have, etc.